bridge/bridgeapp/templates/home.html

```diff
@@ -11,6 +11,7 @@
         <header>
             <nav>
                 <a href="{% url 'home' %} "><img src="/static/images/bridge.png" /></a>
+                <h2>Spanning Questions And Answers</h2>
```

Comment:
I don't understand what you're doing here. I see a <nav> with links lined up horizontally, as I explained here: https://justicethroughcode.slack.com/archives/C03B8N27JG7/p1653113798388519.